spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -219,3 +219,37 @@ Jakarta Data implementations often support the following list of predicate keywo
 |findByNameOrderByAgeAscNameDescYearAsc
 
 |===
+
+===== Special parameter handling
+
+Jakarta Data also supports particular parameters to define either sort or sort.
+
+Jakarta Data recognizes, by default, specific types, like ```Pageable``` and ```Sort```, to dynamically apply pagination and sorting to queries. The following example demonstrates these features:
```

Comment:
Change "by default" to "when specified on a repository method after the query parameters" which will make it easier to figure out which parameters go directly to the database query versus which are for supplying pagination and sorting information.